dotnet/src/SemanticKernel.Core/SemanticKernel.Core.csproj

```diff
@@ -26,6 +26,11 @@
     <InternalsVisibleTo Include="SemanticKernel.Extensions.UnitTests" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection" />
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" />
```

Comment:
```suggestion
```

You don't need both. This comes transitively both through `Microsoft.Extensions.DependencyInjection` and `SemanticKernel.Abstractions.csproj`.